lib/record_store/record/spf.rb

```diff
@@ -10,11 +10,15 @@ def initialize(record)
     end
 
     def to_s
-      "[SPFRecord] #{fqdn} #{ttl} IN SPF \"#{txtdata}\""
+      "[SPFRecord] #{fqdn} #{ttl} IN SPF \"#{rdata_txt}\""
```

Comment:
This shouldn't be necessary because you've put this logic in Record no? https://github.com/Shopify/record_store/pull/44/files#diff-5a4c925834bcd83af90e6868ef7e1db7R67 

I'm guessing you've just missed it :)

Reply:
Same reason as https://github.com/Shopify/record_store/pull/44#discussion_r163387566